tools/topology/topology2/cavs-smart-amp-nocodec.conf

```diff
@@ -0,0 +1,235 @@
+<searchdir:include>
```

Comment:
why is this file not in a test directory? It's at the top level of topology2, this is wrong.